core/src/main/java/software/bernie/geckolib3/core/builder/Animation.java

```diff
@@ -5,9 +5,12 @@
 
 package software.bernie.geckolib3.core.builder;
 
-import java.io.Serializable;
+import java.io.*;
 import java.util.ArrayList;
+import java.util.Base64;
 import java.util.List;
+
+import org.apache.commons.lang3.SerializationUtils;
```

Comment:
Just wanted to ask if using SerializationUtils | ObjectInputStream would cause the [bleeding pipe](https://blog.mmpa.info/posts/bleeding-pipe/) vulnerability?
(although it uses Base64, it would still be nice to clarify)

Reply:
Hey just getting back, this shouldn't be a problem since we aren't using serialization utils or the object input stream class to serialize while sending packets. It's only done internally on the same server/client side to create a deep copy of an animation. In short since it's not being used to send/receive data it's not a problem 👍

Reply:
![image](https://github.com/user-attachments/assets/ac480fe7-3213-44df-afc2-3cb9ff0327e5)

Reply:
Oh, good, thanks for the clarification!